### PR TITLE
Move GPU compilations to installer

### PR DESCRIPTION
**Pull request by dcmvdbekerom**
### Description

This pull request is to move the on-the-fly compilation of GPU codes to the installer.

It addresses the issue where the py_cuffs folder wasn't found, raised [on the Slack channel](https://app.slack.com/client/T9E5R3R9S/C9F7X0ZAT/thread/C9F7X0ZAT-1619738768.112700)

This PR is a "hotfix": the solution is still far from perfect.
Ideally all cupy and cupyx code should be moved out of the cython code into the regular python modules, and only the accelerated CPU routines should be compiled. These could then be moved to the radis_cython_extensions module we already had, and pure-python alternatives should be offered.

### CODE REVIEW

File: radis/lbl/factory.py [hunk 1]
```diff
@@ -993,8 +959,8 @@ def eq_spectrum_gpu(
         log_2vMm = np.array(self._get_log_2vMm(molarmass_arr), dtype=np.float32)
         S0 = np.array(self._get_S0(Ia_arr), dtype=np.float32)
 
-        NwG = 4
-        NwL = 8
+        NwG = 4  # TO-DO: these shouldn't be hardcoded
```
Comment (erwanp) [inline radis/lbl/factory.py:962]: Good that you reported this !

**Review by erwanp — APPROVED** [review]
Great ! Do you think we can also address https://github.com/radis/radis/issues/262#issuecomment-830815789 at the same time ?

**Comment by dcmvdbekerom** [comment]
>Great ! Do you think we can also address #262 (comment) at the same time ?

Yeah should be relatively easy, I'll have a look!

While I had a look at the GPU code, I'm also moving things around a bit to keep only the things that absolutely have to be in cython, in cython. I don't know how long this will take but usually the largest barrier is to get started, which is already overcome :)!

**Comment by dcmvdbekerom** [comment]
Finished moving the GPU code around, now only parts that _have_ to be done in cython are in cython, and they are in the same file as the add_at() function. The rest is in the new lbl/gpu.py file which is much more inviting for future development.

Still haven't looked at #262 yet!

**Comment by erwanp** [comment]
Well done!

**Comment by pkj-m** [comment]
This is some really clean work!! Incredible and definitely LGTM!

**Review by pkj-m — APPROVED** [review]

**Pull request merged**